build/fuses/fuses.json5

```
@@ -7,5 +7,6 @@
   "node_options": "1",
   "node_cli_inspect": "1",
   "embedded_asar_integrity_validation": "0",
-  "only_load_app_from_asar": "0"
+  "only_load_app_from_asar": "0",
+  "load_v8_snapshot_from_custom_path": "0"
```

```suggestion
  "load_browser_process_specific_v8_snapshot": "0"
```

I think that name is better